testing/trino-server-dev/etc/catalog/delta.properties

```diff
@@ -11,6 +11,7 @@ hive.hdfs.socks-proxy=localhost:1180
 
 # MinIO uses 9000 by default, but this change conflicts with Hadoop
 fs.native-s3.enabled=true
+fs.hadoop.enabled=true
```

Comment:
Do we access LocalFS using Hadoop APIs ?

Reply:
We have `hive.hdfs.socks-proxy` config here which doesn't work without enabling hadoop.
It's also convenient for testing against HDFS on local docker containers